src/speeds.jl

```diff
@@ -1,3 +1,27 @@
+export Alfven_velocity, Alfven_speed
+
+function Alfven_velocity(B::BField, ρ::Density)
+    return B / sqrt(μ0 * ρ) |> upreferred
+end
+
+function Alfven_velocity(B::BField, n::NumberDensity; mass_numb = 1, Z = 1)
+    ρ = n * (mass_numb * mp + Z * me)
+    return Alfven_velocity(B, ρ)
+end
+
+Alfven_velocity(B::AbstractVector, args...; kwargs...) =
```

Comment:
This can already be done with broadcasting, so it doesn't need to be specially defined.